utils/check-dependencies-exist.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+'use strict';
+
+const path = require('path');
+const {exec} = require('child_process');
+const glob = require('glob');
+
+const moduleRegex = /modules[\/\w|-]*react/g;
+
+let pass = true;
+
+// Note: Requires dist folders to exist (yarn build to be run)
+glob(`${path.resolve(__dirname)}/../modules/**/react/dist/commonjs/index.js`, undefined, function(
+  er,
+  files
+) {
+  if (er) {
+    console.error('Could not find modules/**/react/dist/commonjs/index.js files');
+    return;
+  }
+
+  const checks = [];
+
+  for (let file of files) {
+    checks.push(
+      new Promise((resolve, reject) => {
+        /* Note: dependency-check can accept multiple files, but due to how it caches imported modules,
+         * it needs to be run separately for each module. */
+        exec(`yarn run dependency-check ${file} --no-dev`, (err, stdout, stderr) => {
+          if (err) {
+            pass = false;
+            const match = file.match(moduleRegex);
+            console.log(match);
```

Comment:
If you want to have an annotation added to PRs (to prevent trying to understand the CLI output message), you could match the ESLint "stylish" output (not all the double-spaces):

```
${fullFilePath}
  ${line}:${column}  error  ${message}  ${type}
```

An example might look like this:
```
/home/runner/work/canvas-kit/canvas-kit/modules/button/react/package.json
  1:1  error  Modules in package.json not used in code: @workday/canvas-kit-react-common  check-dependencies-exist
```

This isn't required, but the annotations are nicer. The error messages appear inline on the `files` tab of a PR rather than digging through the CI logs.